test/views/GovUkWrapperSpec.scala

```diff
@@ -21,6 +21,16 @@ import org.jsoup.nodes.Document
 
 class GovUkWrapperSpec extends ViewBaseSpec {
 
+  "creating a page " should {
```

Comment:
As with the other PR, please can you add the assertion to an existing test as the scenario isn't different